rootfs/etc/cont-init.d/01-adsbhubclient

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/with-contenv sh
+# shellcheck shell=sh
+
+# Check to make sure the correct command line arguments have been set
+EXITCODE=0
+if [ -z "${BEASTPORT}" ]; then
+  echo "BEASTPORT environment variable not set. Using default"
+  export BEASTPORT=30002
```

Comment:
This might not work the way you intend. Exporting here will only be valid for the execution of this script. I can see you reference it in `rootfs/usr/bin/adsbhub.sh`, which will use `30005`; the defined value from Dockerfile. I'd probably remove the export line so all the defaults are defined in the Dockerfile.